ProcessMaker/Http/Controllers/Api/UserController.php

```diff
@@ -140,6 +140,78 @@ public function index(Request $request)
         return new ApiCollection($response);
     }
 
+    /**
+     * Display a listing of users and their task counts.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     *
+     * @OA\Get(
+     *     path="/users_count_tasks",
+     *     summary="Returns all users and their total tasks",
+     *     operationId="getUsersTaskCount",
+     *     tags={"Users"},
+     *     @OA\Parameter(
+     *         name="filter",
+     *         in="query",
+     *         description="Filter results by string. Searches First Name, Last Name, Email, or Username.",
+     *         @OA\Schema(type="string"),
+     *     ),
+     *     @OA\Response(
+     *         response=200,
+     *         description="List of users with task counts",
+     *         @OA\JsonContent(
+     *             type="object",
+     *             @OA\Property(
+     *                 property="data",
+     *                 type="array",
+     *                 ref="#/components/schemas/users",
+     *             ),
+     *             @OA\Property(
+     *                 property="meta",
+     *                 type="object",
+     *                 ref="#/components/schemas/metadata",
+     *             ),
+     *         ),
+     *     ),
+     * )
+     */
+    public function getUsersTaskCount(Request $request)
+    {
+        if (!(Auth::user()->can('view-users') ||
+            Auth::user()->can('create-processes') ||
+            Auth::user()->can('edit-processes') ||
+            Auth::user()->can('create-projects') ||
+            Auth::user()->can('view-projects'))) {
+            throw new AuthorizationException(__('Not authorized to view users.'));
+        }
+        $query = User::nonSystem();
+
+        $filter = $request->input('filter', '');
+        if (!empty($filter)) {
+            $filter = '%' . $filter . '%';
+            $query->where(function ($query) use ($filter) {
+                $query->where('username', 'like', $filter)
+                    ->orWhere('firstname', 'like', $filter)
+                    ->orWhere('lastname', 'like', $filter);
+            });
+        }
+        
+        $query->where('status', 'ACTIVE');
+
+        $subQueryString = "SELECT COUNT(id) FROM process_request_tokens WHERE user_id=users.id AND status='ACTIVE' AND element_type='task'";
+        $query->selectRaw("*, ({$subQueryString}) AS count");
+        $query->groupBy('users.id');
```

Comment:
We should be able to use more query builder functions here. Please look at https://stackoverflow.com/questions/18533080/laravel-eloquent-groupby-and-also-return-count-of-each-group